### PR TITLE
Escape lone surrogates in minimal output

### PR DESCRIPTION
**Pull request by jridgewell**
Re: https://github.com/mathiasbynens/jsesc/pull/59#issuecomment-610537658

### CODE REVIEW

File: src/jsesc.js [hunk 1]
```diff
@@ -241,44 +241,16 @@ const jsesc = (argument, options) => {
...
-			if (character == '`') {
-				result += quote == character ? '\\`' : character;
-				continue;
-			}
-			if (character == '\'') {
-				result += quote == character ? '\\\'' : character;
-				continue;
+			if (character == '"' || character == '`' || character == '\'') {
+				result += quote == character ? '\\' + character : character;
+				continue
```
Comment (michaelficarra) [inline src/jsesc.js:253]:
```suggestion
				continue;
```

File: jsesc.js [hunk 2]
```diff
@@ -241,44 +241,16 @@ const jsesc = (argument, options) => {
...
-				continue;
-			}
-			if (character == '`') {
-				result += quote == character ? '\\`' : character;
-				continue;
-			}
-			if (character == '\'') {
-				result += quote == character ? '\\\'' : character;
-				continue;
+			if (character == '"' || character == '`' || character == '\'') {
```
Comment (michaelficarra) [inline jsesc.js:251]: Wouldn't this be better?

```js
if (character == quote) {
  result += '\\' + quote;
  continue;
}
```

Comment (jridgewell) [reply jsesc.js:251]: That was my original code, but if it's a quote char and not equal to `quote`, we return it without escaping.

**Comment by mathiasbynens** [comment]
Closed by #61.

**Pull request closed without merging**